devel/api.md

```diff
@@ -656,6 +663,46 @@ will also change the interfaces that are visible in the Nmstate state. Reducing
 the total VFs without setting `Interface.STATE` to `InterfaceState.ABSENT` for
 the corresponding interfaces might lead to an undefined system state or error.
 
+### `Ethernet.SRIOV.VFS_SUBTREE`
+
+If `Ethernet.SRIOV.TOTAL_VFS` is 1 or greater, the `Ethernet.SRIOV.VFS_SUBTREE`
+dictionary list allows to configure the VFS special options. Nmstate supports the
+following VF options:
+
+#### `Ethernet.SRIOV.VFS.ID`
+
+Type: `integer`
+
+ID of the VF to configure.
+
+#### `Ethernet.SRIOV.VFS.MAC_ADDRESS`
+
+Type: `string` in the format of `11:22:33:44:55:66`. Case insensitive.
+
+#### `Ethernet.SRIOV.VFS.SPOOF_CHECK`
+
+Type: `boolean`
+
+Enable or disable spoof checking on the VF.
+
+#### `Ethernet.SRIOV.VFS.TRUST`
+
+Type: `boolean`
+
+Enable or disable trust option on the VF.
+
+#### `Ethernet.SRIOV.VFS.MIN_TX_RATE`
+
+Type: `integer`
+
+Set the minimum TX rate on the VF. This value must be greater or equal than 0.
+
+#### `Ethernet.SRIOV.VFS.MAX_TX_RATE`
+
+Type: `integer`
+
+Set the maximum TX rate on the VF. This value must be greater or equal than 0.
```

Comment:
What does `0` mean here? No upper limit or no communication at all? Maybe it could be explained with a link to a SR-IOV reference documentation or we can assume that user know want to set it, know what it means. But what is the default that Nmstate would report?